drafts/FPAAnnualReport2022.md

```diff
@@ -14,72 +14,59 @@ The administrators and members of the FPA would like to thank all of you who put
 
-
 ### Administration team
 
 The FPA administration team expanded to four members in 2022 and is now working at full power. The team maintains a weekly video meeting and no week passes without several tasks being done. Everything learned or set up is being documented in the [FPA handbook](https://fpa.freecad.org/handbook). The handbook covers everything about how the FPA is organized and operated, and serves as a reference for people both inside and outside the FPA.
 
-
 ## Plans for 2023
 
 We expect 2023 to be even busier than 2022.  Some of the things we hope to accomplish in 2023 are:
 
-* Enable the FPA to engage in commercial operations and deduct VAT
-
-* Hire an accountant to handle Belgian reporting requirements, such as tax returns
-
-* Based on 2022 donations, our expected income is €75,000
+* **Enable the FPA to engage in commercial operations** and deduct VAT. The FPA has up to now been exempt from VAT registration, because it operates only on donations. In 2023, we want to enable the FPA to sign contracts and sell services, so it can act as an intermediary and make it easier for companies to contract FreeCAD developers. The VAT registration will enable the FPA to do so, and provide it with a VAT number, which allows it to pay less taxes when buying services and products in the EU.
 
-* Expand the FPADF grant program to a larger amount (€2,000 or €2,500)
+* **Hire an accountant** to handle Belgian reporting requirements, such as tax returns and VAT handling. Up to now, the FPA members have done all the accounting tasks themselves. Hiring an external accountant, however, will guarantee that the FPA does no accounting or tax errors, and will allow the FPA to keep functioning even in case of large changes among FPA members.
 
-* An expanded grants system for tasks proposed by the community, modelled on the Google Summer of Code (GSOC) program
+* Based on 2022 donations and recent trends, **our expected income in 2023 is around €75,000**. A more detailed analysis is available below. A stable income will allow the FPA to launch more ambitious programs, and to engage into more recurring expenses and provide developers more stability.
 
-* Fund other specific projects (open contents, documentation, etc.)
+* **Expand the FPADF grant program** to a larger amount (€2,000 or €2,500). The [FPA's development fund program](https://fpa.freecad.org/programs/fpadf-announcement) has already distributed a few grants in 2022, but not as many as we would have liked. The good financial situation of the FPA will allow us to rise the stakes, and make it more worth for developers to dedicate their time to work on FreeCAD. 
```

Comment:
```suggestion
* **Expand the FPADF grant program** to a larger amount (€2,000 or €2,500). The [FPA's development fund program](https://fpa.freecad.org/programs/fpadf-announcement) has already distributed a few grants in 2022, but not as many as we would have liked. The good financial situation of the FPA will allow us to increase the amount of individual grants to make it more worthwhile for developers to dedicate their time to work on FreeCAD. 
```

Reply:
allow us to **offer larger grants in order to attract more developers to the program**.